tests/test_ilastik/test_workflows/testCarvingGui.py

```diff
@@ -34,14 +34,15 @@
 
 
 # https://bugreports.qt.io/browse/QTBUG-87014
-is_darwin_bigsur = False
+must_skip_darwing_bigsur = False
 if platform.system().lower() == "darwin":
     mac_ver = tuple(map(int, platform.mac_ver()[0].split(".")))
-    if mac_ver >= (10, 6):
-        is_darwin_bigsur = True
+    python_ver = tuple(map(int, platform.python_version().split(".")))
```

Comment:
I wonder if it's time to move this into some `version` utility module.

Reply:
this change introduced some side effects, so with @emilmelnikov, we decided to leave it as is...